src/config.rs

```diff
@@ -363,6 +363,10 @@ macro_rules! cf_config {
                     )
                     .into());
                 }
+                if self.format_version > 4 {
```

Comment:
I think it should be `> 5`. It can just consider whether this version can read it.